### PR TITLE
Change ports for motley-cue

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

`motley-cue` uses `8080`, like `traefik` for the dashboard. This PR moves `motley-cue` to a different port.

---

**Related issue :**

### CODE REVIEW

File: deployment/main.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ resource "openstack_compute_instance_v2" "dashboard" {
   network {
     uuid = var.net_id
   }
-  security_groups = [openstack_compute_secgroup_v2.secgroup.name, "default"]
+  security_groups = ["default", "HTTP", "motley-cue"]
```
Comment (enolfc) [inline deployment/main.tf:9]: should we remove `default`?

**Review by enolfc — APPROVED** [review]
LGTM

File: deployment/main.tf (hunk 1 above)
Comment (sebastian-luna-valero) [reply deployment/main.tf:9]: `ssh` is there, should I add it to `motley-cue` then?

File: README.md [hunk 2]
```diff
@@ -13,12 +13,14 @@ This code relies on docker-compose to run 3 containers:
...
-accessible host with a valid name:
+accessible host with a valid name. You can create a `.env` file with the
+`DASHBOARD_HOSTNAME` variable defined with the hostname of your server and just
+start the service:
 
 ```shell
 cd /path/to/working/directory
 git clone https://github.com/EGI-Federation/fedcloud-dashboard.git
-cd fedcloud-dashboard
+echo "DASHBOARD_HOSTNAME="<your host name>"
```
Comment (enolfc) [inline README.md:23]:
```suggestion
echo "DASHBOARD_HOSTNAME="<your host name>" > .env
```

File: deployment/main.tf (hunk 1 above)
Comment (enolfc) [reply deployment/main.tf:9]: yes, I think it's better to have explicit control of the ports than to rely on groups that we don't control

**Review by enolfc — CHANGES_REQUESTED** [review]
just 2 minor things

File: deployment/main.tf (hunk 1 above)
Comment (sebastian-luna-valero) [reply deployment/main.tf]: done, thanks

**Review by enolfc — APPROVED** [review]
:+1:

**Pull request merged**